other/upgrade.php

```diff
@@ -76,7 +76,7 @@
 
 // The helper is crucial. Include it first thing.
 if (!file_exists($upgrade_path . '/upgrade-helper.php'))
-    die('upgrade-helper.php not found where it was expected: ' . $upgrade_path . '/upgrade-helper.php! Make sure you have uploaded ALL files from the upgrade package. The upgrader cannot continue.');
+	die('upgrade-helper.php not found where it was expected: ' . $upgrade_path . '/upgrade-helper.php! Make sure you have uploaded ALL files from the upgrade package. The upgrader cannot continue.');
```

Comment:
Why did you added extra tab here? I think its unnecessary

Reply:
Look more closely ;)